src/pages/starter-kit/checkout/tax-use-cases.md

```diff
@@ -204,3 +204,35 @@ Here's an example payload showing how the custom attributes from tax classes app
   }
 }
 ```
+## Propagate serialized custom attributes
+
+### Tax class custom attributes available in Quote
+
+The `oope-tax-management` module introduces support for assigning serialized custom attributes to tax classes. These attributes are then automatically associated with shopping cart data during cart creation and product addition. This allows for tax-related metadata to be included early in the checkout process and carried forward into subsequent operations.
+
+### How it works
+
+- **Customer Tax Class → Quote**:  
+  When a customer creates a shopping cart, any serialized custom attributes associated with the customer's tax class are copied to the `Quote` entity.
+
+- **Product Tax Class → Quote Item**:  
+  When a product is added to the cart, serialized custom attributes from the product's tax class are copied to the corresponding `Quote Item`.
+
+This ensures that both the `Quote` and each `Quote Item` contain tax-relevant custom data for further processing or integration.
+
+---
```

Comment:
I don't see any reference to using `---` in this repository, so I would suggest removing it. 
Generally, when writing in a specific style or sentences, I recommend reviewing the existing documents and aligning with their writing style, as writers are likely to request changes otherwise.